### PR TITLE
docs: Trusted collaborator onboarding checklist

### PR DESCRIPTION
**Pull request by manughub**
Checklist for onboarding trusted collaborators. Copy the contents of this file to your onboarding PRs.

## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: ONBOARDING_CHECKLIST [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Onboarding checklist for Trusted collaborators
+
+Copy this to your onboarding PRs.
+
+
+- [ ] Get a NDA signed [Manu]
```
Comment (kara) [inline ONBOARDING_CHECKLIST]:
```suggestion
- [ ] Get an NDA signed [Manu]
```

File: ONBOARDING_CHECKLIST [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+- [ ] Add to collaborators@angular.io distribution [Manu]
+- [ ] Add to <<sub-team>>@angular.io distribution [Manu]
+- [ ] invite these groups (and not individuals) to the regular sync meetings [Lead]
+- [ ] Add these groups to the meeting notes & other docs [Lead]
+- [ ] Add to Jira [Manu]
+- [ ] Add to Slack [Manu]
+- [ ] Add to required slack channels [Mentor]
```
Comment (kara) [inline ONBOARDING_CHECKLIST]: Where's the guidance on which channels these should be? Is there a master list?

File: ONBOARDING_CHECKLIST [hunk 3]
```diff
@@ -0,0 +1,18 @@
+# Onboarding checklist for Trusted collaborators
```
Comment (kara) [inline ONBOARDING_CHECKLIST]: Can we put this file in the docs folder so it's not at root?

**Review by kara — APPROVED** [review]
LGTM

File: docs/ONBOARDING_CHECKLIST [hunk 4]
```diff
@@ -0,0 +1,18 @@
+# Onboarding Checklist for Trusted Collaborators
+
+Copy this to your onboarding PRs.
```
Comment (gkalpak) [inline docs/ONBOARDING_CHECKLIST]: What are on-boarding PRs? :grin:

File: docs/ONBOARDING_CHECKLIST [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+
+
+- [ ] Get an NDA signed [Manu]
+- [ ] Add to collaborators@angular.io distribution [Manu]
+- [ ] Add to <<sub-team>>@angular.io distribution [Manu]
+- [ ] invite these groups (and not individuals) to the regular sync meetings [Lead]
```
Comment (gkalpak) [inline docs/ONBOARDING_CHECKLIST]: Is `Lead` the lead of the sub-team that the collaborator will be primarily working with?

File: docs/ONBOARDING_CHECKLIST (hunk 4 above)
Comment (manughub) [reply docs/ONBOARDING_CHECKLIST]: @gkalpak PRs for onboarding collaborators. Here's an example
https://github.com/angular/angular/pull/29554

File: docs/ONBOARDING_CHECKLIST (hunk 5 above)
Comment (manughub) [reply docs/ONBOARDING_CHECKLIST]: Yes sub-team lead.

File: docs/ONBOARDING_CHECKLIST (hunk 4 above)
Comment (gkalpak) [reply docs/ONBOARDING_CHECKLIST]: I see, thx!

File: docs/ONBOARDING_CHECKLIST (hunk 5 above)
Comment (IgorMinar) [inline docs/ONBOARDING_CHECKLIST]: I don't think that this item is part of the onboarding checklist. This is an item for onboarding of subteams which is different. Please remove

File: docs/ONBOARDING_CHECKLIST [hunk 6]
```diff
@@ -0,0 +1,18 @@
+# Onboarding Checklist for Trusted Collaborators
+
+Copy this to your onboarding PRs.
+
+
+- [ ] Get an NDA signed [Manu]
```
Comment (IgorMinar) [inline docs/ONBOARDING_CHECKLIST]:
```suggestion
- [ ] Get an NDA signed [Manu]
- [ ] Accept [Angular Communication Guidelines](http://goo.gl/p7BoeV) [Collaborator]
- [ ] Read and get familiar with [the Angular Mission](https://g.co/ng/mission) [Collaborator]
```

File: docs/ONBOARDING_CHECKLIST (hunk 5 above)
Comment (IgorMinar) [inline docs/ONBOARDING_CHECKLIST]:
```suggestion
```

File: docs/ONBOARDING_CHECKLIST [hunk 7]
```diff
@@ -0,0 +1,18 @@
...
+- [ ] Add to collaborators@angular.io distribution [Manu]
+- [ ] Add to <<sub-team>>@angular.io distribution [Manu]
+- [ ] invite these groups (and not individuals) to the regular sync meetings [Lead]
+- [ ] Add these groups to the meeting notes & other docs [Lead]
```
Comment (IgorMinar) [inline docs/ONBOARDING_CHECKLIST]:
```suggestion
```

this is for sub-group onboarding, not individual onboarding.

File: docs/ONBOARDING_CHECKLIST (hunk 4 above)
Comment (IgorMinar) [inline docs/ONBOARDING_CHECKLIST]: From this instruction it's not clear if you want people to copy the MD file (if so, where this file should be stored) or if you want people to copy the contents of the MD file into the pull request description.

I'd prefer the former (copy the file) eventually, and have one file per person with info like profile url, (speaker) bio, etc. But I'm fine with for now saying that people should just copy the contents to PR description and we can improve the process later and ask already onboarded people to backfill the missing info in followup PRs.

**Review by IgorMinar — APPROVED** [review]
a few clarifications, but otherwise lgtm

**Comment by IgorMinar** [comment]
unless there are issues with my suggestions, please resolve all pending comments and merge. thanks

File: docs/ONBOARDING_CHECKLIST (hunk 4 above)
Comment (manughub) [reply docs/ONBOARDING_CHECKLIST]: I was thinking they would copy the contents of this file into the onboarding PR description.

File: docs/ONBOARDING_CHECKLIST (hunk 6 above)
Comment (manughub) [reply docs/ONBOARDING_CHECKLIST]: How are we going to verify they have done these ? Acceptance email for communication guidelines ? Mentor introduces them the mission statement ?

File: docs/ONBOARDING_CHECKLIST (hunk 4 above)
Comment (kara) [reply docs/ONBOARDING_CHECKLIST]: Nit: We may want to move the `copy this` instruction above the checklist header, so it's easier to copy-paste the title + checklist together.

**Comment by IgorMinar** [comment]
This PR is now obsolete as we use a slightly different process for onboarding. This process uses a google doc.

**Pull request closed without merging**